site/content/en/_index.html

```diff
@@ -55,16 +55,16 @@ <h2 class="text-center pb-3">Companies using Agones</h2>
 		<a href="http://www.yager.de/"><img alt="yager" width="100" src="images/yagerdevelopment.png" /></a>
 		<a href="http://mcmahan.games/"><img alt="mcmahan.games" width="100" src="images/mcmahangames.png" /></a>
 		<a href="http://altavr.io/"><img alt="altavr" width="100" src="images/altavr-logo.png" /></a>
-		<a href="https://www.embark.games/" data-proofer-ignore><img alt="Embark Studios" width="50" src="images/embark.png" /></a>
+		<a href="https://www.embark.games/"><img alt="Embark Studios" width="50" src="images/embark.png" /></a>
 		<a href="http://accelbyte.io/"><img alt="accelbyte" width="100" src="images/accelbyte.png" /></a>
 		<a href="https://space.game/"><img alt="Space Game" width="80" src="images/spacegame.png" /></a>
-		<a href="https://rolltable.app/" data-proofer-ignore><img alt="rolltable" width="100" src="images/rolltable.png" /></a>
-		<a href="https://vela.games/" data-proofer-ignore><img alt="Vela Games" width="100" src="images/velagames.png" /></a>
+		<a href="https://rolltable.app/"><img alt="rolltable" width="100" src="images/rolltable.png" /></a>
+		<a href="https://vela.games/"><img alt="Vela Games" width="100" src="images/velagames.png" /></a>
 		<a href="https://netspeakgames.com//"><img alt="Netspeak Games" width="100" src="images/netspeakgames.png" /></a>
-		<a href="https://afterverse.com/" data-proofer-ignore><img alt="Afterverse" width="100" src="images/afterverse.svg" /></a>
-		<a href="https://rocketbotroyale.winterpixel.io/" data-proofer-ignore><img alt="Winterpixel Games" width="100" src="images/winterpixel.png" /></a>
-		<a href="https://vizor-games.com" data-proofer-ignore><img alt="Vizor Games" width="100" src="images/vizor-games.png" /></a>
-		<a href="https://acceleratxr.com" data-proofer-ignore><img alt="AcceleratXR" width="100" src="images/acceleratxr.png" /></a>
```

Comment:
I wonder if this will pass with this turned back on. With `link_test` and `data-proofer-ignore` we generally only look at .md files -- but it will see if these fail for some reason being turned back on, and can revert if we need to.

Reply:
Done.